### PR TITLE
feat:added pages

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**
I have created E-commerce pages 
1. Home
2.productDisplay
3. Cart
4. Wish
5. Auth

### CODE REVIEW

File: src/Authentication/login.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            className="btn btn-link brand"
+          >
+            Venice
+          </a>
+        </div>
+        <div className="right">
+          <div className="input-group search">
+            <input type="text" placeholder="Search" />
+          </div>
+          <a href="https://venice-app.netlify.app/Authentication/login.html">
```
Comment (mansipadgelwar) [inline src/Authentication/login.js]: Change anchor tags to Link

**Review by mansipadgelwar — APPROVED** [review]
Good to merge

File: src/Authentication/login.js (hunk 1 above)
Comment (KamiyaGaikwad) [reply src/Authentication/login.js]: Thank you for pointing it out. I corrected it

**Review by vinc3nati — APPROVED** [review]
Nice structured and concise code

**Pull request merged**